color_refinement.py

```diff
@@ -221,18 +221,21 @@ def get_number_automorphisms(g: Graph) -> int:
 
     graphs = L[0]
     print("Graph: ", GRAPH)
+    automorphisms = {}
     for i in range(len(graphs)):
         for j in range(len(graphs)):
             if j == i:
                 start = time.time()
                 num = get_number_automorphisms(graphs[i])
-                print('There are', num, 'automorphisms')
+                automorphisms[graphs[i].name] = num
+                print('Graph', graphs[i].name, 'has', num, 'automorphisms')
                 print('Took', time.time() - start, 'seconds\n')
             if j > i:
                 start = time.time()
                 isomorph = is_isomorphisms(graphs[i], graphs[j])
-                print(graphs[i].name,'and',graphs[j].name,'isomorphic?',isomorph)
-                coloring = initialize_coloring(graphs[i]+graphs[j])
-                num = count_isomorphism(graphs[i], graphs[j],coloring)
-                print('There are',num,'isomorphisms')
-                print('Took',time.time()-start,'seconds\n')
+                if isomorph:
```

Comment:
Er moet wel een `else` zijn, want de timer is nu gestart en stopt mogelijk niet meer.

Het boeit me niet dat we niet zien dat twee grafen _niet_ isomorf zijn, maar het timen van het bepalen van isomorfismen moet wel goed blijven gaan.

Reply:
Volgens mij wordt die in de huidige situatie ook niet gestopt, want time.time() is niet echt het stoppen van een timer. Daarnaast starten we ook niet echt een timer. We nemen gewoon een tijdstip `start = time.time()` en trekken daar een andere tijdstip vanaf. Dus dit hoeft niet